Framework/DataHandling/src/SaveNXcanSAS.cpp

```diff
@@ -24,6 +24,16 @@ void SaveNXcanSAS::init() { initStandardProperties(); }
 
 std::map<std::string, std::string> SaveNXcanSAS::validateInputs() { return validateStandardInputs(); }
 
+bool SaveNXcanSAS::checkGroups() {
+  Mantid::API::Workspace_sptr &&workspace = getProperty("InputWorkspace");
```

Comment:
I don't think we gain anything by making this an r-value reference. If `getProperty("InputWorkspace");` returns an r-value then `Mantid::API::Workspace_sptr workspace = getProperty("InputWorkspace");` will use the move constructor to avoid a copy, which is efficient enough.

Reply:
It may be even better to do `const Mantid::API::Workspace_sptr &workspace = getProperty("InputWorkspace");`, as the `isGroup` method is marked const. This will work as you can assign r-values to const l value references.